packages/radix/src/components/Popover.tsx

```diff
@@ -8,14 +8,16 @@ import { Card } from './Card';
 
 type PopoverProps = PopoverPrimitiveProps & {
   isOpen?: boolean;
+  showArrow?: boolean;
 };
 
-export const Popover = ({ children, ...props }: PopoverProps) => {
+export const Popover = ({ children, showArrow, ...props }: PopoverProps) => {
   return (
     <PopoverPrimitive
       side="bottom"
       align="center"
-      arrow={<Arrow width={20} height={10} />}
+      // TODO: Arrow `null` to not render an Arrow Component
+      arrow={showArrow ? <Arrow width={20} height={10} /> : <div />}
```

Comment:
Yeah in Popover primitive, we default to using our own Arrow if we don't get an arrow prop. I haven't redone that primitive yet, but I get the feeling a bunch of stuff is gonna change…